v1/lib/core/renderMarkdown.js

```diff
@@ -45,14 +47,20 @@ class MarkdownRenderer {
                 siteConfig.usePrism.length > 0 &&
                 siteConfig.usePrism.indexOf(lang) !== -1)
             ) {
+              const language = alias[lang] || lang;
               try {
-                const language = alias[lang] || lang;
                 // Currently people using prismjs on Node have to individually require()
                 // every single language (https://github.com/PrismJS/prism/issues/593)
                 require(`prismjs/components/prism-${language}.min`);
                 return prismjs.highlight(str, prismjs.languages[language]);
               } catch (err) {
-                console.error(err);
+                if (err.code === 'MODULE_NOT_FOUND') {
+                  const unsupportedLanguageError = chalk.yellow(
+                    `Warning: language-${language} is not supported by prismjs.` +
```

Comment:
Let's omit the `language-` so that it will be something like

```
`Warning: ${language} is not supported by prismjs.`
```

Even better if we put different color for the language itself

Reply:
@endiliey Okay, Ive made that change.